OrbitGl/CaptureSerializer.cpp

```diff
@@ -78,6 +79,11 @@ void CaptureSerializer::FillCaptureData(CaptureInfo* capture_info) {
     capture_info->add_address_infos()->CopyFrom(address_info.second);
   }
 
+  const absl::flat_hash_map<uint64_t, FunctionStats> functions_stats =
```

Comment:
maybe const reference here?

Reply:
You are totally right. I missed the ref. Changed it accordingly. Thanks.